salsah/src/public/js/jquery.resadd.js

```diff
@@ -517,6 +517,7 @@
 												}
 											}
 											tmpele.attr('placeholder', placeholderText);
+											tmpele.attr('autocomplete', 'off');
```

Comment:
oh man, I'm sorry to push this one late, but it is a little refinement that helps; turning autocomplete off makes that there is no drop down list of previous attempts that hides the "official list":

![capture d ecran 2017-06-26 a 11 13 40](https://user-images.githubusercontent.com/3933654/27532653-437e445a-5a61-11e7-87db-944eb421b0de.png)